File: mlt.common/domainEntities/RssFeed.cs
namespace mlt.common.domainEntities;

public class RssFeed
{
    public string? Id { get; set; }

    public string Name { get; set; } = null!;

    public string Url { get; set; } = null!;

    public DateTime LastUpdate { get; set; }

    public string Category { get; set; } = null!;

    public string DestinationFolder { get; set; } = null!;
}

File: mlt.web/src/app/core/models/RssFeed.ts
import {RssFeedResult} from "./RssFeedResult";

export interface RssFeed {
  id?: string; // Optional property
  name: string;
  url: string;
  lastUpdate: Date | undefined; // Use Date type for DateTime
  forceFirstSeasonFolder: boolean;
  fileNameRegex : string;
  category: string;
  destinationFolder: string;
  results: RssFeedResult[];
}
